robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -37,7 +37,37 @@ def click_actions_button(self,button_name):
             self.selenium.click_link(button_name)
         else:
             self.selenium.click_button(button_name)
-    
+
+    @capture_screenshot_on_error
+    def edit_recurring_donation(self,**kwargs):
+        """From the actions dropdown select edit action and edit the fields specified in the kwargs"""
+        locator=npsp_lex_locators['bge']['button'].format("Edit")
+        edit_button=self.selenium.get_webelement(locator)
+        self.selenium.wait_until_page_contains_element(edit_button, error="Show more actions dropdown didn't open in 30 sec")
```

Comment:
I feel it's not a good idea to hardcode "30 seconds", since the global timeout might change.
Its okay to hardcode if you added a "timeout" argument here in this keyword.